modules/ROOT/pages/about-data-types.adoc

```diff
@@ -1,4 +1,5 @@
 = Define and Use Data Types in a Flow (Flow Designer)
+:page-support-status: eolReached 
```

Comment:
Should this be `eolScheduled`? The EOL dashboard says Flow Designer reaches End of Standard Support on 2/15/2024.

Reply:
I was checking that with the PM. Yes, I thought that the EOL banner was "scheduled".

Reply:
Specifically `eolScheduled`. Gary updated the wiki:
https://confluence.internal.salesforce.com/display/MTDT/Using+Banners#UsingBanners-DocsThatAreScheduledtoReachEnd-Of-Life(EOL)

Reply:
Addressed: https://github.com/mulesoft/docs-design-center/pull/206/commits/9ead98c8ce603f763b4a2cf3cf871ec557c4b827